src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,89 @@
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+        >
+          «
+        </a>
+      </li>
+
+      {Array(numberOfPages)
+        .fill(null)
+        .map((_, index) => {
+          return (
+            <li
+              key={index + 1}
+              className={cn('page-item', {
+                active: currentPage === index + 1,
+              })}
+            >
+              <a
+                data-cy="pageLink"
+                className="page-link"
+                href="#1"
+                onClick={e => {
+                  e.preventDefault();
+                  onPageChange(index + 1);
+                }}
+              >
+                {index + 1}
+              </a>
+            </li>
+          );
+        })}
+
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === numberOfPages,
+          active: currentPage === numberOfPages,
+        })}
+      >
+        <a
+          data-cy="nextLink"
+          className="page-link"
+          href="#next"
```

Comment:
The `href` attribute for the next link is hardcoded to `#next`. If you are preventing the default action and using onClick to handle the page change, you might not need the `href` attribute at all, or you could use `href='#!'` to indicate that it's a placeholder for a dynamic link.